packages/orbit-components/src/FeatureIcon/index.tsx

```diff
@@ -24,20 +22,14 @@ const generateURL = (name: string) => {
   return { src: urlWithName(), srcSet: [urlWithName(2), urlWithName(3)].join(",") };
 };
 
-export const StyledFeatureIcon = styled(({ className, name, alt, dataTest, id }) => (
-  <img className={className} alt={alt} data-test={dataTest} id={id} {...generateURL(name)} />
-))`
-  height: ${baseSize}px;
-  width: auto;
-  background-color: transparent;
-`;
-
-StyledFeatureIcon.defaultProps = {
-  theme: defaultTheme,
-};
-
 const FeatureIcon = ({ alt = "", name, dataTest, id }: Props) => (
-  <StyledFeatureIcon alt={alt} name={name} dataTest={dataTest} id={id} />
+  <img
+    className="h-[52px] w-auto bg-transparent"
```

Comment:
I don't know if this height class is necessary 🤔 but if anything, we can also probably use the constant directly. Defining a variable above, something like `const heightClass = `h-[${baseSize}px]`;` and then use it here with template string.
I am not sure if it is the best practice in Tailwind, but it uses a constant that won't change in runtime, so it can be built in build time and we can rely on it. We're also safe for possible changes on the `baseSize` constant value.

But, again, I am not sure if it's a good practice and I won't enforce it if you're against it, so it's up to you.

Reply:
Even if I don't like so much interpolating variables into Tailwind classes, I believe it's safer to reuse `baseSize` to cover us from side-effects when changing this value. My bad that I didn't realize and hardcoded the value instead.
I'll follow your suggestion.